bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment/Atmospheric/Earth/CSSISpaceWeather.cpp

```diff
@@ -1,5 +1,7 @@
 /// Apache License 2.0
 
+#include <pybind11/functional.h>
```

Comment:
We need this because we are passing a function as arg in `access_last_reading_where` correct?

Reply:
Yes 👍